addons/account/models/account_move_line.py

```diff
@@ -473,7 +473,7 @@ def get_name(line):
             elif line.journal_id.type == 'purchase':
                 if product.description_purchase:
                     values.append(product.description_purchase)
-            return '\n'.join(values)
+            return '\n'.join(values) if line else False
```

Comment:
```suggestion
            if not product:
                return False
```
Thanks for the fix. :+1: 

You can add this after computing the product as it is done in `saas-17.4`

@robodoo up to saas-17.2

Reply:
Yes sir, done, please review :heart_eyes: